soccer/Processor.cpp

```diff
@@ -532,99 +499,14 @@ void Processor::sendRadioData() {
     // Add RadioTx commands for visible robots and apply joystick input
```

Comment:
Don't forget to update this comment

Reply:
Changed the name of the function instead